components/Navbar.tsx

```diff
@@ -21,8 +21,14 @@ import MenuIcon from "@mui/icons-material/Menu";
 import CategoryDropDownMenu from "./CategoryDropDownMenu";
 
 const drawerWidth = 240;
-const navItems = ["Home", "Sign In", "Donation", "Mission"];
-const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page"];
+const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard"];
+const links = [
+  "/",
+  "/auth/sign-in",
+  "/donation-info",
+  "/mission-page",
+  "dashboard",

```

Comment:
There seems to be a routing error due to a missing forward slash "/". Adding the forward slash should resolve the issue.